projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -18,6 +18,7 @@
     "cy:run:ci:ssr": "cypress run --config-file cypress.ci.json --config baseUrl=http://localhost:4000 --record --key $CYPRESS_KEY --tag \"ssr,all\" --parallel --group SSR --spec \"cypress/integration/ssr/pages.e2e-spec.ts\"",
     "cy:run:ci:ccv2": "cypress run --config-file cypress.ci.json --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API --spec \"cypress/integration/cx_ccv2/regression/b2c/**/*.e2e-spec.ts\"",
     "cy:run:ci:ccv2-b2b": "cypress run --config-file cypress.ci.json --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API,BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --spec \"cypress/integration/cx_ccv2/regression/b2b/**/*.e2e-spec.ts\"",
+    "cy:run:ci:ccv2-cpq": "cypress run --config-file cypress.ci.json --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API,BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --spec \"cypress/integration/**/cpq-configuration.ccv2-e2e-spec.ts\"",
```

Comment:
where will this be tested on ccv2? My assumption from the thread that you dm'd me is for p3 right?

If it's for p3, is the base site really powertools-spa or is `powertools-epdvisualization-spa` as done in the package script `cy:run:ci:epd-visualization`

Reply:
Yes we want to test on P3.
We test on powertools-spa